tests/gemmini/conv/test_gemmini_conv.py

```diff
@@ -1,448 +0,0 @@
-from __future__ import annotations
```

Comment:
Why was this file deleted?

Reply:
This is a super outdated test that was skipped (see line 8). It's fine not to delete this file, but it doesn't matter

Reply:
Ok, I was just making sure it was intentional.